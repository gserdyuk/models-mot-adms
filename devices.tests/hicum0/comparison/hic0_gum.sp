HICUM0 Gummel Test Ic=f(Vc,Vb)

VB B 0 0.5
VC C 0 1.0
VS S 0 0.0
X1 C B 0 S DT hicumL0V1p1_c_sbt

.control
dc vb 0.2 1.4 0.01
run
*plot abs(i(vc)) abs(i(vb)) abs(i(vs)) ylimit 0.1e-12 100e-3 ylog
*plot abs(i(vc))/abs(i(vb)) vs abs(-i(vc)) xlog xlimit 1e-09 10e-3 ylimit 0 300
*print v(b) v(c) v(s) v(dt) i(vb) i(vc) i(vs) 
print  i(vb) i(vc) i(vs) 
.endc

.include ../model-card-hicumL0V1p11.lib

.end
