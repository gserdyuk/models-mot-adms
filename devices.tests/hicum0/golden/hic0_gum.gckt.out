Gnucap 2009.12.07 RCS 26.136
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
Q: already installed, replacing
stashing as Q:0
HICUM0 Gummel Test Ic=f(Vc,Vb) 
.model hic0_ful ... =1.0e-10  tnom=27.0 dt=0.0 npn=1 pnp=0 
                                        ^ ? hic0_full: bad parameter dt ignored
#           i(VB)      i(VC)      i(VS)     
 0.2        0.         0.         1.E-12    
 0.21       0.         0.         1.E-12    
 0.22       0.         0.         1.E-12    
 0.23       0.         0.         1.E-12    
 0.24       0.         0.         1.E-12    
 0.25       0.         0.         1.E-12    
 0.26       0.         0.         1.E-12    
 0.27       0.         0.         1.E-12    
 0.28       0.         0.         1.E-12    
 0.29       0.         0.         1.E-12    
 0.3        0.         0.         1.E-12    
 0.31       0.         0.         1.E-12    
 0.32       0.         0.         1.E-12    
 0.33       0.         0.         1.E-12    
 0.34       0.         0.         1.E-12    
 0.35       0.         0.         1.E-12    
 0.36       0.         0.         1.E-12    
 0.37       0.         0.         1.E-12    
 0.38       0.         0.         1.E-12    
 0.39       0.         0.         1.E-12    
 0.4        0.         0.         1.E-12    
 0.41       0.         0.         1.E-12    
 0.42       0.         0.         1.E-12    
 0.43       0.         0.         1.E-12    
 0.44       0.         0.         1.E-12    
 0.45       0.         0.         1.E-12    
 0.46       0.         0.         1.E-12    
 0.47       0.         0.         1.E-12    
 0.48       0.         0.         1.E-12    
 0.49       0.         0.         1.E-12    
 0.5        0.         0.         1.E-12    
 0.51       0.         0.         1.E-12    
 0.52       0.         0.         1.E-12    
 0.53       0.         0.         1.E-12    
 0.54       0.         0.         1.E-12    
 0.55       0.         0.         1.E-12    
 0.56       0.         0.         1.E-12    
 0.57       0.         0.         1.E-12    
 0.58       0.         0.         1.E-12    
 0.59       0.        -11.458E-9  1.E-12    
 0.6        0.        -16.842E-9  1.E-12    
 0.61       0.        -24.758E-9  1.E-12    
 0.62       0.        -36.404E-9  1.E-12    
 0.63       0.        -53.524E-9  1.E-12    
 0.64       0.        -78.715E-9  1.E-12    
 0.65       0.        -115.7E-9   1.E-12    
 0.66       0.        -170.13E-9  1.E-12    
 0.67       0.        -250.11E-9  1.E-12    
 0.68       0.        -367.71E-9  1.E-12    
 0.69       0.        -540.61E-9  1.E-12    
 0.7        0.        -794.73E-9  1.E-12    
 0.71       0.        -1.1683E-6  1.E-12    
 0.72      -9.6013E-9 -1.7172E-6  1.E-12    
 0.73      -13.997E-9 -2.5236E-6  1.E-12    
 0.74      -20.413E-9 -3.7081E-6  1.E-12    
 0.75      -29.794E-9 -5.4469E-6  1.E-12    
 0.76      -43.482E-9 -7.9975E-6  999.9E-15 
 0.77      -63.454E-9 -11.735E-6  999.9E-15 
 0.78      -92.519E-9 -17.203E-6  999.9E-15 
 0.79      -134.75E-9 -25.186E-6  999.8E-15 
 0.8       -195.96E-9 -36.801E-6  999.7E-15 
 0.81      -284.29E-9 -53.625E-6  999.5E-15 
 0.82      -410.99E-9 -77.832E-6  999.3E-15 
 0.83      -591.25E-9 -112.35E-6  999.E-15  
 0.84      -844.76E-9 -160.96E-6  998.5E-15 
 0.85      -1.196E-6  -228.33E-6  997.9E-15 
 0.86      -1.6735E-6 -319.77E-6  997.1E-15 
 0.87      -2.308E-6  -440.84E-6  996.E-15  
 0.88      -3.1295E-6 -596.65E-6  994.5E-15 
 0.89      -4.1639E-6 -791.15E-6  992.8E-15 
 0.9       -5.4304E-6 -0.0010266  990.6E-15 
 0.91      -6.9404E-6 -0.0013033  988.1E-15 
 0.92      -8.698E-6  -0.00162    985.2E-15 
 0.93      -10.702E-6 -0.0019742  981.9E-15 
 0.94      -12.948E-6 -0.0023628  978.4E-15 
 0.95      -15.428E-6 -0.0027823  974.5E-15 
 0.96      -18.136E-6 -0.0032294  970.4E-15 
 0.97      -21.064E-6 -0.0037008  966.1E-15 
 0.98      -24.206E-6 -0.0041938  961.6E-15 
 0.99      -27.557E-6 -0.0047058  956.9E-15 
 1.        -31.111E-6 -0.0052345  952.E-15  
 1.01      -34.864E-6 -0.0057782  947.1E-15 
 1.02      -38.813E-6 -0.006335   941.9E-15 
 1.03      -42.955E-6 -0.0069036  936.7E-15 
 1.04      -47.288E-6 -0.0074828  931.4E-15 
 1.05      -51.81E-6  -0.0080715  926.E-15  
 1.06      -56.52E-6  -0.0086687  920.6E-15 
 1.07      -61.415E-6 -0.0092736  915.E-15  
 1.08      -66.495E-6 -0.0098856  909.4E-15 
 1.09      -71.76E-6  -0.010504   903.7E-15 
 1.1       -77.208E-6 -0.011128   898.E-15  
 1.11      -82.84E-6  -0.011758   892.2E-15 
 1.12      -88.654E-6 -0.012392   886.4E-15 
 1.13      -94.651E-6 -0.013032   880.6E-15 
 1.14      -100.83E-6 -0.013675   874.7E-15 
 1.15      -107.19E-6 -0.014322   868.7E-15 
 1.16      -113.74E-6 -0.014973   862.8E-15 
 1.17      -120.46E-6 -0.015627   856.8E-15 
 1.18      -127.37E-6 -0.016285   850.8E-15 
 1.19      -134.46E-6 -0.016945   844.7E-15 
 1.2       -141.74E-6 -0.017608   838.6E-15 
 1.21      -149.19E-6 -0.018274   832.5E-15 
 1.22      -156.83E-6 -0.018942   826.4E-15 
 1.23      -164.66E-6 -0.019612   820.3E-15 
 1.24      -172.66E-6 -0.020285   814.1E-15 
 1.25      -180.86E-6 -0.020959   807.9E-15 
 1.26      -189.23E-6 -0.021635   801.7E-15 
 1.27      -197.8E-6  -0.022314   795.5E-15 
 1.28      -206.54E-6 -0.022993   789.3E-15 
 1.29      -215.48E-6 -0.023675   783.E-15  
 1.3       -224.6E-6  -0.024358   776.8E-15 
 1.31      -233.91E-6 -0.025042   770.5E-15 
 1.32      -243.41E-6 -0.025728   764.2E-15 
 1.33      -253.1E-6  -0.026415   757.9E-15 
 1.34      -262.98E-6 -0.027103   751.6E-15 
 1.35      -273.05E-6 -0.027793   745.3E-15 
 1.36      -283.32E-6 -0.028483   739.E-15  
 1.37      -293.78E-6 -0.029175   732.6E-15 
 1.38      -304.44E-6 -0.029867   726.3E-15 
 1.39      -315.31E-6 -0.03056    719.9E-15 
 1.4       -326.39E-6 -0.031254   713.6E-15 
